frontend/packages/kubevirt-plugin/package.json

```diff
@@ -9,6 +9,10 @@
     "@console/shared": "0.0.0-fixed",
     "kubevirt-web-ui-components": "~0.1.35"
   },
+  "scripts": {
+    "lint": "cd ../.. && eslint --ext .js,.jsx,.ts,.tsx --color packages/kubevirt-plugin",
+    "lintfix": "yarn lint --fix"
```

Comment:
I prefer `lint:fix`. 
However recent scripts use `lint-fix` convention so we should stay aligned.

Reply:
I decided to leave only `lint` and one can use `lint --fix` like in the main package.json